spec/features/cart_view_spec.rb

```diff
@@ -6,10 +6,12 @@
       FactoryGirl.create(:status, id: 1)
       @item = FactoryGirl.create(:item)
       user = FactoryGirl.create(:user)
-      visit login_path
-      fill_in("session_email", with: user.email)
-      fill_in("session_password", with: user.password)
-      click_button 'Sign in'
+      visit root_path
+      within("#signInModal") do
+        fill_in("session_email", with: user.email)
+        fill_in("session_password", with: user.password)
+        click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.